src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer is starting its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer has stopped working.");
```

Comment:
Similarly, the message in `stopWork` method should also be more informative. It should clearly indicate which machine has stopped working.